common/style/main.css

```diff
@@ -0,0 +1,240 @@
+@charset "UTF-8";
+
+/* CSS Document */
+@import url(http://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600,300italic);
+
+body { 
+	padding: 0;
+	margin: 0;
+	font-family: 'Source Sans Pro', Arial, sans-serif;
+}
+
+.section-padding {
+    padding: 60px 0;
+}
+
+.section-header {
+	font-size: 42px;
+	display: block;
+	text-align: center;
+	margin-bottom: 24px;
+}
+
+.section-header-text {
+	display: block;
+	text-align: center;
+	margin: 12px 0 36px;
+}
+
+
+
+/**
+ * Header Styles...
+ */
+
+#header {
+    background:
+    	url("/common/images/shane-warrior.png") bottom left no-repeat,
+    	url("/common/images/half-dome.jpg") bottom center no-repeat;
+    background-size:
+    	325px,
+    	cover;
+    display: block;
+    min-height: 450px;
+}
+
+.shanespiration {
+    float: left;
+    background-image: url("/common/images/logo-white.png");
+    background-size: 225px 50px;
+    background-repeat: no-repeat;
+    padding-top: 12px;
+    padding-left: 225px;
+    margin-top: 24px;
+    height: 50px;
+    display: block;
+}
+
+.centeredInspire {
```

Comment:
@paulstanleyjohnson any reason for the camelCase vs the hyphenation naming convention? Not sure if this was there already from the friend of Col's that previous made the site...my bad if so!